atomisticparsers/gromacs/parser.py

```diff
@@ -840,7 +841,10 @@ def parse_thermodynamic_data(self):
 
         if not thermo_data:
```

Comment:
Isn't this completely redundant? We already checked if the edr files has keys to parse